frontend/src/pages/NewProject/AutoJoinInput.module.css

```diff
@@ -0,0 +1,21 @@
+.container {
+	align-items: flex-start;
+	row-gap: var(--size-xxSmall);
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	max-height: fit-content;
+	min-height: var(--size-xxLarge);
+}
+
+.select {
+	&:global(.ant-select-disabled .ant-select-selector) {
+		background-color: transparent !important;
+	}
+
+	:global(.ant-select-selector) {
+		padding: 0 !important;
+		border: 0px solid var(--color-gray-300) !important;
+		background-color: transparent !important;
+	}
+}
```

Comment:
i'm guessing this syntax might be causing the discrepancy with reflame? see https://github.com/highlight/highlight/pull/7254

Reply:
I think the select looks inline with vite and reflame 🤔 but can take another look.

Lewis did some research on some the discrepancies [here](https://highlightcorp.slack.com/archives/C04DVJJ0ZJ6/p1703025239265259), and agree that I think this is a regression on the vite side of things, and reflame is handling our css better (correctly?)